stl/src/StlCompareStringA.cpp

```diff
@@ -18,7 +18,8 @@
 //
 // Entry:
 //        LPCWSTR LocaleName  - locale context for the comparison.
-//        DWORD   dwCmpFlags  - see docs.microsoft.com
+//        DWORD   dwCmpFlags  - see:
+//                    https://learn.microsoft.com/en-us/windows/win32/api/stringapiset/nf-stringapiset-comparestringex

```

Comment:
😕 
Can this be formatted better?

Reply:
This is the best I could do with a 120 column limit, unless you have a better idea. (One occurrence could technically have been unwrapped but I felt it was best to make them all consistently wrapped.)

Reply:
* "see `CompareStringEx` in Platform SDK documentation"
* an URL shortener, if there's one that can be trusted, like from microsoft.com domain

Reply:
I do actually have the ability to create aka ms links, I can do that here.

Reply:
`https://learn.microsoft.com/en-us/windows/win32/api/stringapiset/nf-stringapiset-comparestringex`
is equivalent to:
`learn.microsoft.com/windows/win32/api/stringapiset/nf-stringapiset-comparestringex`

14 chars less.